seam/seam.py

```diff
@@ -10,7 +10,38 @@
 
 class Seam(AbstractSeam):
     """
-    Initial Seam class used to interact with Seam API
+    Main class for interacting with the Seam API.
+
+    This class provides methods to authenticate and interact with various Seam API endpoints,
+    including devices, access codes, action_attempts, and more. It supports authentication via API key or personal access token.
+
+    Attributes:
+    ----------
+        lts_version (str): The long-term support (LTS) version of the Seam Python SDK.
+        defaults (Dict[str, Any]): Default settings for API requests.
+        client (SeamHttpClient): The HTTP client used for making API requests.
+        wait_for_action_attempt (Union[bool, Dict[str, float]]): Controls whether to wait for an action attempt to complete.
+
+    Inherited Attributes from Routes class:
+    --------------------------------
+        access_codes (AccessCodes): Interface for access code-related operations.
+        acs (Acs): Interface for access control system operations.
+        action_attempts (ActionAttempts): Interface for action attempt operations.
+        client_sessions (ClientSessions): Interface for client session operations.
+        connect_webviews (ConnectWebviews): Interface for connect webview operations.
+        connected_accounts (ConnectedAccounts): Interface for connected account operations.
+        devices (Devices): Interface for device-related operations.
+        events (SeamEvents): Interface for event-related operations.
+        locks (Locks): Interface for lock-related operations.
+        networks (Networks): Interface for network-related operations.
+        noise_sensors (NoiseSensors): Interface for noise sensor operations.
+        phones (Phones): Interface for phone-related operations.
+        thermostats (Thermostats): Interface for thermostat operations.
+        user_identities (UserIdentities): Interface for user identity operations.
+        webhooks (Webhooks): Interface for webhook operations.
+        workspaces (Workspaces): Interface for workspace operations.
```

Comment:
If we are going to document this, we need to make sure it's part of codegen. But do we need this here? Wouldn't' the docs for these attributes be at the attribute / method level?

Reply:
I wasn't sure about adding these so happy to move them to attribute / method level

Reply:
a9109042009db96f2351ccaf21bc3ab5dea67ad1